RecDP/pyrecdp/primitives/operations/text_ingestion.py

```diff
@@ -99,6 +105,7 @@ class BaseDocumentIngestion(BaseLLMOperation, ABC):
     def __init__(self,
                  text_column: str = 'text',
                  embeddings_column: str = 'embedding',
+                 update: bool = False,
```

Comment:
maybe rename  'update' to 'vector_persist_mode' for better understanding? them use valid values: 'append' or 'override' ?